pkgs/tools/networking/requestly/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, appimageTools, fetchurl }:
+
+let
+  pname = "requestly";
+  version = "1.5.6";
+
+  src = fetchurl {
+    url =
+      "https://github.com/requestly/requestly-desktop-app/releases/download/v${version}/Requestly-${version}.AppImage";
+    hash = "sha256-Yb90OGIIvExfNPoJPmuZSvtU5OQVuGqh4EmyKltE+is=";
+  };
+
+  appimageContents = appimageTools.extractType2 { inherit pname version src; };
+
+in
+appimageTools.wrapType2 {
+  inherit pname version src;
+
+  extraInstallCommands = ''
+    mv $out/bin/${pname}-${version} $out/bin/${pname}
+
+    install -Dm 444 ${appimageContents}/${pname}.desktop -t $out/share/applications
```

Comment:
This also needs a 
```
    substituteInPlace $out/share/applications/${pname}.desktop \
      --replace 'Exec=AppRun' 'Exec=${pname}'
```
to fix the .desktop file

Reply:
Does it? Since there is no `Exec=AppRun` line in the .desktop file at all ->

```
[Desktop Entry]
Name=Requestly
Exec=requestly --no-sandbox %U
Terminal=false
Type=Application
Icon=requestly
StartupWMClass=Requestly
X-AppImage-Version=1.5.6
Comment=Intercept & Modify HTTP Requests
MimeType=x-scheme-handler/requestly;
Categories=Development;
```

Reply:
Interesting, for me (NixOS x86_64-linux, nix 2.25.1) it is. 
```
# result/share/applications/requestly.desktop
[Desktop Entry]
Name=Requestly
Exec=AppRun --no-sandbox %U
Terminal=false
Type=Application
Icon=requestly
StartupWMClass=Requestly
X-AppImage-Version=1.5.6
Comment=Intercept & Modify HTTP Requests
MimeType=x-scheme-handler/requestly;
Categories=Development;
```